projects/igniteui-angular/src/lib/grids/common/crud.service.ts

```diff
@@ -97,10 +97,30 @@ export class IgxCell {
         public rowIndex: number,
         public column,
         public value: any,
-        public editValue: any,
+        public _editValue: any,
         public rowData: any,
         public grid: GridType) {
         this.grid.validation.create(id.rowID, rowData);
+        this.editValue = this._editValue;
```

Comment:
This will likely trigger an extra validation, though I think I saw the public API creating this with different value/editValue initially, so we might be stuck with this.